code/modules/projectiles/guns/ballistic/rifle.dm

```diff
@@ -123,6 +123,10 @@
 	mag_type = /obj/item/ammo_box/magazine/internal/boltaction/harpoon
 	fire_sound = 'sound/weapons/gun/sniper/shot.ogg'
 	can_be_sawn_off = FALSE
+	can_jam = FALSE
+	jamming_chance = 0
+	unjam_chance = 0
+	jamming_increment = 0
```

Comment:
Surely can_jam = FALSE should be enough? If not, the jamming code is bad.

Reply:
It should but I thought I'd hit these anyway.

Reply:
I'd rather you didn't, it lies about the stability of can_jam 😄

Reply:
fair